modules/github_integration/lib/open_project/github_integration/patches/api/work_package_representer.rb

```diff
@@ -13,6 +13,13 @@ def extension
                 title: "github"
               }
             end
+
+            link :github_pull_requests do
+              {
+                href: api_v3_paths.github_pull_requests_by_work_package(represented.id),
+                title: "github_pull_requests"
```

Comment:
The title is typically intended to be human readable as it might be displayed in a client.